src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs

```diff
@@ -7,48 +7,27 @@
 
 namespace System.Net.Sockets
 {
-    /// <devdoc>
-    ///    <para>
-    ///       The <see cref='System.Net.Sockets.UdpClient'/> class provides access to UDP services at a
-    ///       higher abstraction level than the <see cref='System.Net.Sockets.Socket'/> class. <see cref='System.Net.Sockets.UdpClient'/>
-    ///       is used to connect to a remote host and to receive connections from a remote
-    ///       Client.
-    ///    </para>
-    /// </devdoc>
+    // The System.Net.Sockets.UdpClient class provides access to UDP services at a higher abstraction
+    // level than the System.Net.Sockets.Socket class. System.Net.Sockets.UdpClient is used to
+    // connect to a remote host and to receive connections from a remote client.
     public class UdpClient : IDisposable
     {
         private const int MaxUDPSize = 0x10000;
+
         private Socket _clientSocket;
         private bool _active;
         private byte[] _buffer = new byte[MaxUDPSize];
-        /// <devdoc>
-        ///    <para>
-        ///       Address family for the client, defaults to IPv4.
-        ///    </para>
-        /// </devdoc>
         private AddressFamily _family = AddressFamily.InterNetwork;
 
-        // bind to arbitrary IP+Port
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Net.Sockets.UdpClient'/>class.
-        ///    </para>
-        /// </devdoc>
+        // Initializes a new instance of the System.Net.Sockets.UdpClientclass.
         public UdpClient() : this(AddressFamily.InterNetwork)
```

Comment:
Nit: For TcpClient you've moved the `: this(...)` on the next line. Which one should we use?